examples/restaurant_demo/restaurant.py

```diff
@@ -0,0 +1,308 @@
+            .astype(int)
+        )
+
+        # Use the default epsilon value if it is not given as an argument
+        if not epsilon:
+            x = dp.CountInt(self._epsilon)
+        else:
+            x = dp.CountInt(epsilon)
+
+        for time in visits["Time entered"].unique():
+            private_hours_count[time] = x.result(
+                list(visits[visits["Time entered"] == time]["Time entered"].astype(int))
+            )
+
+        return private_hours_count
+
+    def get_non_private_counts_per_day(self) -> dict:
+        """Compute the number of visits per day without any differential privacy. Return a dictionary mapping days to number of visits"""
+        day_counts = dict()
+
+        for day in self._day_visits["Day"].unique():
+            day_counts[day] = self._day_visits[self._day_visits["Day"] == day][
+                "Day"
+            ].count()
+
+        return day_counts
+
+    def get_private_counts_per_day(self, epsilon: float = None) -> dict:
+        """Compute an anonymized (within a given threshold of epsilon) version of the number of visits per day. Return a dictionary mapping days to number of visits"""
+        # Pre-process the data set: limit the number of days contributed by a visitor to COUNT_MAX_CONTRIBUTED_DAYS
+        day_visits = bound_visits_per_week(self._day_visits, COUNT_MAX_CONTRIBUTED_DAYS)
+
+        day_counts = dict()
+
+        # Use the default epsilon value if it is not given as an argument
+        if not epsilon:
+            x = dp.CountInt(self._epsilon)
+        else:
+            x = dp.CountInt(epsilon)
+
+        for day in day_visits["Day"].unique():
+            day_counts[day] = x.result(
+                list(day_visits[day_visits["Day"] == day]["Day"])
+            )
+
+        return day_counts
+
+    def get_non_private_sum_revenue(self) -> dict:
+        """Compute the revenue per day of visits without any differential privacy. Return a dictionary mapping days to revenue"""
+        day_revenue = dict()
+
+        for day in self._day_visits["Day"].unique():
+            day_revenue[day] = self._day_visits[self._day_visits["Day"] == day][
+                "Money spent (euros)"
+            ].sum()
+
+        return day_revenue
+
+    def get_private_sum_revenue(self, epsilon: float = None) -> dict:
+        """Compute an anonymized (within a given threshold of epsilon) version of the revenue per day. Return a dictionary mapping days to revenue"""
+        # Pre-process the data set: limit the number of days contributed by a visitor to SUM_MAX_CONTRIBUTED_DAYS
+        day_visits = bound_visits_per_week(self._day_visits, SUM_MAX_CONTRIBUTED_DAYS)
+
+        day_revenue = dict()
+
+        # Use the default epsilon value if it is not given as an argument
+        if not epsilon:
+            x = dp.BoundedSum(self._epsilon, MIN_EUROS_SPENT, MAX_EUROS_SPENT_1)
+        else:
+            x = dp.BoundedSum(epsilon, MIN_EUROS_SPENT, MAX_EUROS_SPENT_1)
+
```

Comment:
Here also l0 sensitivity needs to be set (to SUM_MAX_CONTRIBUTED_DAYS).

It's a [method call](https://github.com/google/differential-privacy/blob/9923ad4ee1b84a7002085e50345fcc05f2b21bcb/examples/java/SumRevenuePerDay.java#L100) in Java:
```java
              .maxPartitionsContributed(MAX_CONTRIBUTED_DAYS)
```